sdk/confidentialledger/confidential-ledger-rest/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.0.2 (Unreleased)
+
+### Features Added
+
+- User List endpoint
```

Comment:
Is there a reason this is a patch version since this is adding new features?  In addition, should this be a beta release since you are using a preview version of the API?

Reply:
Hi @mpodwysocki, yes you're right on both counts, I'll fix the versioning.  

How do I mark this release as a beta release?  Is it just changing the version in the CHANGELOG to something like `1.1.0-beta.1`?  

Thanks @mpodwysocki

Reply:
Yes, that would be correct, `1.1.0-beta.1` would be the correct version.

Reply:
Thanks @mpodwysocki , the version has been changed to `1.1.0-beta.1` everywhere for this release.